fe/fe-core/src/main/java/com/starrocks/sql/analyzer/mvpattern/MVColumnBitmapUnionPattern.java

```diff
@@ -64,13 +64,7 @@ public boolean match(Expr expr) {
             if (!child0FnExpr.getFnName().getFunction().equalsIgnoreCase(FunctionSet.TO_BITMAP)) {
                 return false;
             }
-            SlotRef slotRef = child0FnExpr.getChild(0).unwrapSlotRef();
```

Comment:
why remove this check?

Reply:
Now we can support non-slot ref expressions, eg to_bitmap(a * 100)